src/shared/command.h

```diff
@@ -349,6 +351,7 @@ static inline const char *escapeid(ident &id) { return escapeid(id.name); }
 extern bool validateblock(const char *s);
 extern char *parsetext(const char *&p);
 extern void explodelist(const char *s, vector<char *> &elems, int limit = -1);
```

Comment:
That prototype should no longer be needed, right?

Reply:
Could we flag it with a notice that says "deprecated, please use new `std::vector` version"? I think doxygen has a syntax for that.